C23_WMDE_Mobile_DE_04

moiikana opened the pull request:
based on VAR 02, 
has all payment types as text, 
VAR leads to different address page

https://phabricator.wikimedia.org/T345530

tzhelyazkova commented on shared/progress_bar/style.pcss:
```diff
@@ -87,7 +87,7 @@
 	}
 }
 
-@media ( max-width: 900px ) {
```
This is changed due to progress bar's left side text disappearing on small iPad sizes.
Therefor now this text should only be hidden for mobile. This might be changed in the future.

moiikana committed:
Archive C23_WMDE_Mobile_DE_03

https://phabricator.wikimedia.org/T345530

moiikana committed:
Prepare campaign for C23_WMDE_Mobile_DE_04

- based on VAR vor Mobile 02 (not 03)

https://phabricator.wikimedia.org/T345530

moiikana committed:
Create VAR for C23_WMDE_Mobile_DE_04

- form leads to different AB test bucket (address page)
- change tests

https://phabricator.wikimedia.org/T345530
